cmd/commands/git-source.go

```diff
@@ -189,53 +199,67 @@ func NewGitSourceCreateCommand() *cobra.Command {
 				GsName:              args[1],
 				RuntimeName:         args[0],
 				CreateDemoResources: false,
+				Include:             include,
+				Exclude:             exclude,
+				Flow:                flow,
 			})
 		},
 	}
 
 	cmd.Flags().BoolVar(&createRepo, "create-repo", false, "If true, will create the specified git-source repo in case it doesn't already exist")
+	cmd.Flags().StringVar(&include, "include", "", "include")
+	cmd.Flags().StringVar(&exclude, "exclude", "", "exclude")
 
 	insCloneOpts = apu.AddCloneFlags(cmd, &apu.CloneFlagsOptions{CloneForWrite: true})
 	gsCloneOpts = apu.AddCloneFlags(cmd, &apu.CloneFlagsOptions{
 		Prefix:   "git-src",
 		Optional: true,
 	})
 
+	flow = store.Get().GsCreateFlow
+
 	return cmd
 }
 
 func RunGitSourceCreate(ctx context.Context, opts *GitSourceCreateOptions) error {
-	// upsert git-source repo
-
-	gsRepo, gsFs, err := opts.GsCloneOpts.GetRepo(ctx)
+	version, err := getRuntimeVersion(ctx, opts.RuntimeName)
 	if err != nil {
-		return fmt.Errorf("failed to clone git-source repo: %w", err)
+		return err
 	}
 
-	if opts.CreateDemoResources {
-		if err := createDemoResources(ctx, opts, gsRepo, gsFs); err != nil {
-			return fmt.Errorf("failed to create git-source demo resources: %w", err)
-		}
+	if opts.Flow == store.Get().InstallationFlow {
```

Comment:
why use a `Flow string` with two possible values? wouldn't it be simpler to use some bool flag called `ForceAutopilot` or something like that? (`ForceLocal`? )

Reply:
I thought it might come in handy in future cases, to differentiate between the two flows like we need to do in this case. no?